src/pages/About/index.js

```diff
@@ -1,5 +1,36 @@
-import { Wrapper } from "./style";
-
+import { Card, CardsContainer, HighlightText, Wrapper } from "./style";
+import ComunnityImg from '../../assets/community.svg';
+import EmpowermentImg from '../../assets/empowerment.svg';
+import LightBoxImg from '../../assets/light-box.svg';
 export const About = () => {
-  return <Wrapper>About</Wrapper>;
+  return (
+    <Wrapper>
+          <h2>SOBRE NÓS</h2>
```

Comment:
É uma boa prática de acessibilidade deixar da seguinte forma:

```suggestion
          <h2>Sobre nós</h2>
```

Isso para que leitores de tela para pessoas com deficiências leiam o conteudo de forma correta. Ao invés de colocar o texto em capslock no html é melhor fazer uso do `text-transform: uppercase` no CSS do elemento

Reply:
Outra coisinha, como você vai alterar o estilo do h2, cria uma tag no arquivo ./style.js `Title` que tem as propriedades que você quer alterar, a vantagem disso é que o CSS de modificação do h2 fica todo encapsulado dentro da tag `Title` 😄